readme.md

```diff
@@ -1,72 +1,32 @@
-
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo and extract it into the Kodi Add-on folder.
+There are two ways to install Retrospect, depending on what version of Kodi you are using:
 
-#### Caution!
-Be aware that if you install it using *Option A* or *Option B* Kodi will **disable** the add-on by default and it will **not install any of the dependencies**. So you need to enable it manually and install all the dependencies by hand. Starting from Kodi Leia this is easier as it has a "View Dependencies" option of add-ons.   
+### Kodi Leia and later
+Starting from Kodi Leia (v18), you can easy install Retrospect via the Kodi repository. Either search the add-ons in the Kodi repository for Retrospect and install it. More detailed information can be found on the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
+**Note:** Make sure to enable the _Auto-Update_ option for Retrospect to automatically receive new updates.
 
-### 3 - Updating Retrospect ###
+![alt text](./plugin.video.retrospect/resources/media/retrospect01.jpg "The Retrospect information screen")
 
-#### Option A - ....via 'GIT Clone' ####
-Pull latest changes into your clone (located in the Kodi Add-on folder). After that remove all existing `*.pyc` and `*.pyo` files within the Retrospect folders (**don't skip this**).
+### Kodi Krypton    
+If you are running Kodi Krypton (v17) the only way to install Retrospect is to install it from a release zip file. Follow these steps (keep in mind they might slightly differ depending on your skin):
 
-#### Option B - ....via 'Full Zip Download' ####
-Download the complete GIT repo. Remove all existing Retrospect folders (**don't skip this**) and extract the new ones it into the Kodi Add-on folder.
+1. Download the latest release zip file from here: [https://github.com/retrospect-addon/plugin.video.retrospect/releases](https://github.com/retrospect-addon/plugin.video.retrospect/releases)
+1. Transfer the file to you Kodi system (_optional_)
+1. Install Retrospect from zip file
 
-#### Finalizing the update ####
-In both situation run Retrospect at least once before accessing the Retrospect add-on settings. The initial run might take longer than usual, as Retrospect is initialising some stuff and downloads artwork.
+More detailed information can be found on the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-# ! Be advised ! #
-Retrospect will NOT auto-update. So new version need to be installed manually. 
+**Note:** Keep in mind, that because Retrospect is not in the Kodi repository it will not automagically update. Retrospect will notifiy you of new versions. You will then have to manually update you Retrospect add-on with the steps aboven.
 
```

Comment:
**Note:** Since Retrospect isn't available in the official Kodi add-on repository for Kodi 17 Krypton, it will not automagically update. Retrospect will notifiy you of new versions. You will then have to manually update you Retrospect add-on using the steps above.

Reply:
✔️

Reply:
Can't comment anything under Troubleshooting below. Section removed?

Reply:
Nothing changed there. This was specifically a PR for the installation related documentation.